test/jdk/javax/swing/JScrollBar/4865918/bug4865918.java

```diff
@@ -43,24 +46,15 @@ public class bug4865918 {
     public static void main(String[] argv) throws Exception {
         try {
             Robot robot = new Robot();
-            SwingUtilities.invokeAndWait(new Runnable() {
-
-                public void run() {
-                    createAndShowGUI();
-                }
-            });
+            SwingUtilities.invokeAndWait(() -> createAndShowGUI());
 
             robot.waitForIdle();
+            robot.delay(1000);
 
-            SwingUtilities.invokeAndWait(new Runnable() {
-
-                @Override
-                public void run() {
-                    sbar.pressMouse();
-                }
-            });
+            SwingUtilities.invokeAndWait(() -> sbar.pressMouse());
 
             robot.waitForIdle();
+            robot.delay(200);
 
             int value = getValue();
```

Comment:
`getValue()` can be evaluated in if condition directly, no need to have `value` variable.

typo in incorrect
throw new Error("The scrollbar block increment is `incorrect`");

may be changed to RTE instead of Error.

Reply:
ok